src/intention/SectorExitRepositoryFactory.cpp

```diff
@@ -65,34 +53,37 @@ namespace UKControllerPlugin {
+            points["NALAN"] = std::make_unique<SectorExitPoint>("NALAN", "R2", SectorExitPoint::outNorth);
+            points["OSBON"] = std::make_unique<SectorExitPoint>("OSBON", "R3", SectorExitPoint::outNorth);
+            points["PEMOS"] = std::make_unique<SectorExitPoint>("PEMOS", "R5", SectorExitPoint::outNorth);
+            points["RIXUN"] = std::make_unique<SectorExitPoint>("RIXUN", "R6", SectorExitPoint::outNorth);
+            points["SOSAR"] = std::make_unique<SectorExitPoint>("SOSAR", "R7", SectorExitPoint::outNorth);
+            points["GUNPA"] = std::make_unique<SectorExitPoint>("GUNPA", "R8", SectorExitPoint::outNorth);
 
             // Shanwick
-            points["ATSIX"] = std::make_unique<SectorExitPoint>("ATSIX", "Y", SectorExitPoint::outWest);
-            points["BALIX"] = std::make_unique<SectorExitPoint>("BALIX", "N9", SectorExitPoint::outWest);
-            points["ERAKA"] = std::make_unique<SectorExitPoint>(
-                "ERAKA",
-                "N8",
-                SectorExitPoint::outWest | SectorExitPoint::outNorthWest
-            );
-
-            // ADODO isn't really an exit point, but it's near ERAKA
-            points["ADODO"] = std::make_unique<SectorExitPoint>(
-                "ADODO",
-                "N8",
-                SectorExitPoint::outWest | SectorExitPoint::outNorthWest
-            );
-            points["GOMUP"] = std::make_unique<SectorExitPointShanwick>("GOMUP", "N7", SectorExitPoint::outWest);
-            points["IBROD"] = std::make_unique<SectorExitPoint>("IBROD", "N", SectorExitPoint::outWest);
-            points["MIMKU"] = std::make_unique<SectorExitPointShanwick>("MIMKU", "N6", SectorExitPoint::outWest);
-            points["NIBOG"] = std::make_unique<SectorExitPointShanwick>("NIBOG", "N5", SectorExitPoint::outWest);
+            points["LUSEN"] = std::make_unique<SectorExitPoint>("LUSEN", "Y8", SectorExitPoint::outNorthWest | SectorExitPoint::outNorth);
+            points["ATSIX"] = std::make_unique<SectorExitPoint>("ATSIX", "Y7", SectorExitPoint::outWest | SectorExitPoint::outNorthWest);
+            points["ORTAV"] = std::make_unique<SectorExitPoint>("ORTAV", "Y6", SectorExitPoint::outWest | SectorExitPoint::outNorthWest);
+            points["BALIX"] = std::make_unique<SectorExitPoint>("BALIX", "Y5", SectorExitPoint::outWest | SectorExitPoint::outNorthWest);
+            points["ADODO"] = std::make_unique<SectorExitPoint>("ADODO", "Y4", SectorExitPoint::outWest | SectorExitPoint::outNorthWest);
+            points["ERAKA"] = std::make_unique<SectorExitPoint>("ERAKA", "Y3", SectorExitPoint::outWest | SectorExitPoint::outNorthWest);
+            points["ETILO"] = std::make_unique<SectorExitPoint>("ETILO", "Y2", SectorExitPoint::outWest | SectorExitPoint::outNorthWest);
+			
+			// Shanwick NOTA
+            points["GOMUP"] = std::make_unique<SectorExitPointShanwick>("GOMUP", "Y1", SectorExitPoint::outWest | SectorExitPoint::outNorthWest);

```

Comment:
This will cause UKCP to display the sector exit as `U1` if the route is via Belfast VOR. Is this intended behaviour (note, any SEP using `SectorExitPointShanwick` will be changed to UX, where X is the number if via BEL)?